src/common/components/button/button.module.scss

```diff
@@ -0,0 +1,14 @@
+@import "colors";
+@import 'variables';
+
+.buttonWrapper {
+    display: flex;
```

Comment:
Not sure why flex is needed here

Reply:
i dont think grid is needed all the time

Reply:
Yeah, I think neither `flex` nor `grid` is needed here.

Reply:
agree, i just want to make sure it can take all from parents